spring-cloud-task-batch/src/test/java/org/springframework/cloud/task/batch/partition/DeployerPartitionHandlerTests.java

```diff
@@ -344,23 +344,27 @@ public void testPassingEnvironmentProperties() throws Exception {
 		AppDeploymentRequest request = this.appDeploymentRequestArgumentCaptor.getValue();
 
 		assertEquals(this.resource, request.getResource());
-		assertEquals(2, request.getDeploymentProperties().size());
-		assertEquals("bar", request.getDeploymentProperties().get("foo"));
-		assertEquals("qux", request.getDeploymentProperties().get("baz"));
+		assertEquals(2, request.getDefinition().getProperties().size());
+		assertEquals("bar", request.getDefinition().getProperties().get("foo"));
+		assertEquals("qux", request.getDefinition().getProperties().get("baz"));
 
 		AppDefinition appDefinition = request.getDefinition();
 
 		assertEquals("partitionedJobTask:partitionedJob:step1:partition1", appDefinition.getName());
-		assertEquals("1", appDefinition.getProperties().get(DeployerPartitionHandler.SPRING_CLOUD_TASK_JOB_EXECUTION_ID));
-		assertEquals("4", appDefinition.getProperties().get(DeployerPartitionHandler.SPRING_CLOUD_TASK_STEP_EXECUTION_ID));
-		assertEquals("step1", appDefinition.getProperties().get(DeployerPartitionHandler.SPRING_CLOUD_TASK_STEP_NAME));
+		assertTrue(request.getCommandlineArguments().contains(formatArgs(DeployerPartitionHandler.SPRING_CLOUD_TASK_JOB_EXECUTION_ID, "1")));
```

Comment:
duh sorry - and all of the `assertEquals` look correct... I was blurring 2 sides of the diff in my mind I guess... carry on ;)